sources/api/migration/migrations/v1.9.0/image-gc-thresholds/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "image-gc-thresholds"
+version = "0.1.0"
+edition = "2018"
+authors = ["Mahdi Chaker <mmchaker@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+publish = false
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[dependencies]
+migration-helpers = { path = "../../../migration-helpers", version = "0.1.0"}
```

Comment:
I'm not sure if any values here should be customized. 
E.g.: should 2018 be 2022?

Reply:
Update: 2018 should stay 2018 -- it's a Rust thing (ecosystem version)